app/helpers/kuroko2/job_definitions_helper.rb

```diff
@@ -34,5 +34,59 @@ def star_link_for(definition)
         )
       end
     end
+
+    def format_kuroko_script(script_text)
+      raw(
```

Comment:
[nits IMO] If contents are mutli lines, I think appending `.html_safe` is simpler.